Sources/UITextField.swift

```diff
@@ -48,12 +48,15 @@ extension UITextField {
       }.disposeIn(rBag)
       
       self.rControlEvent
-        .filter { $0 == UIControlEvents.EditingChanged }
+        .filter { $0 == UIControlEvents.AllEditingEvents }
         .observeNext { [weak self] event in
           guard let unwrappedSelf = self else { return }
-          updatingFromSelf = true
-          unwrappedSelf.rText.value = unwrappedSelf.text
-          updatingFromSelf = false
+          // only send to rText if value changed, as .AllEditingEvents reports more than just changes
+          if unwrappedSelf.rText.value != unwrappedSelf.text {
```

Comment:
Is there any way to avoid this? I don't like the idea of performing string comparison O(n) on each keystroke...